gotpm/cmd/read.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	pb "github.com/golang/protobuf/proto"
+	"github.com/google/go-tpm-tools/tpm2tools"
+	"github.com/spf13/cobra"
+)
+
+var readCmd = &cobra.Command{
+	Use:   "read <pcr>",
+	Short: "Read from the TPM",
+	Long:  `Read from the TPM`,
+	Args:  cobra.NoArgs,
+}
+
+var pcrCmd = &cobra.Command{
+	Use:   "pcr",
+	Short: "Read PCRs from the TPM",
+	Long: `Read PCRs from the TPM
+
+Based on the --pcrs flag, this reads the contents of the TPM's PCRs.
```

Comment:
I would rephrase this to clarify that there is always _some_ PCR selection and always _some_ `hashAlgo` selection.

Something like:
```
Read PCRs from the TPM

Read the contents of each PCR specified with --pcrs. If --pcrs is not provided,
all pcrs in the bank are read.

This command only supports reading from a single PCR bank. The bank can be
changed using the --hash-algo flag.
```